jarm/scanner/scanner.py

```diff
@@ -99,9 +107,16 @@ async def scan_async(
                 for r in result_list:
                     if p[0] == r[0]:
                         results.append(Scanner._parse_server_hello(r[1], p))
+        except OSError:
```

Comment:
What error do you want to track here? 

We already do some checks here https://github.com/PaloAltoNetworks/pyjarm/blob/a83c13b57c497bb0b1daf8dfdfbc0e81cec1464e/jarm/validate/validate.py#L12 and we raise a PyJARMInvalidTarget if the target is invalid.

Timeouts error should trigger an `asyncio.TimeoutError` Exception. Timeout considers connection time + proxy time (if present) + waiting for response time (we set pass the timeout when we wait for the connection to be handled and the results to be provided).

A timeout of a single connection should result in a TOTAL_FAILURE?

Reply:
I think I can remove this, I was concerned about other uncaught socket exceptions, but doing some more reading it sounds like `socket.gaierror` is the only one expected to arise when using `getaddrinfo`.

I guess we are doing as much checking as is reasonable upstream, and so maybe the blanket `except Exception` here is sufficient.

Reply:
Unless we want to differentiate reporting for timeouts or handle scenarios of single TCP connection failure differently than TOTAL_FAILURE, I think the broader Exception catch good enough to solve the problem.